CONTRIBUTING.md

```diff
@@ -9,29 +9,27 @@
 
 If you want to submit an issue and you want your issue to be resolved quickly, here's a checklist for you:
 
-- Read the [Manual](https://github.com/ccxt/ccxt/wiki/Manual), and especially carefully read the following sections:
+- Ensure you are using recent version of ccxt.
+- Read the [Manual](https://github.com/ccxt/ccxt/wiki/Manual), and especially these sections:
   - [Exchange Properties](https://github.com/ccxt/ccxt/wiki/Manual#exchange-properties)
```

Comment:
Markdown formatting doesn't work well on all systems if we don't separate paragraphs with empty lines.

Reply:
ah, i didn't know those systems. should I close PR?

Reply:
Yeah, i'd close it for now, will add those minor edits there later.